src/socket-message-handling.ts

```diff
@@ -119,6 +128,29 @@ const setupWSConnection = (conn) => {
           case "ping":
             send(conn, { type: "pong" });
         }
+      } else if (Buffer.isBuffer(message)) {
+        // handle websocket connection
+        const doc = getYDoc(docName, gc);
+        const encoder = encoding.createEncoder()
+        const decoder = decoding.createDecoder(message)
+        const messageType = decoding.readVarUint(decoder)
+        switch (messageType) {
+          case messageSync:
+            encoding.writeVarUint(encoder, messageSync)
+            syncProtocol.readSyncMessage(decoder, encoder, doc, null)
+            if (encoding.length(encoder) > 1) {
+              doc.send(conn, encoding.toUint8Array(encoder))
+            }
+            // TODO:
+            // doc is now the updated document
+            // we need to figure out how to persist this
+            break
```

Comment:
one we get the message sync events, we should be able to pull out the contents of the document with:

```
doc.getText('codemirror').toString()
```

We should make sure to persist the entire contents of 
```
doc.getText('codemirror').toJSON()
```

because I think that has the whole change history, which we need to hold onto